apps/i18n/netsim/en_ploc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "yes": "Yes"
```

Comment:
@tanya can tell you more about pseudolocalization (repeating myself to see if hipchat picks up the @mention).